### PR TITLE
fix(docker): include globally installed puppeteer

### PR DESCRIPTION
**Pull request by Eliasvdb**
I noticed puppeteer was not included in the lhci-client docker image. I altered the `Dockerfile` to include a global npm install of puppeteer. I set the ENV var `PUPPETEER_SKIP_DOWNLOAD=true`, because otherwise the puppeteer install will try to download chromium (but it will not have permission). As far as I tested this chrmium step is not needed as the docker image itself contains `google-chrome-stable`.

### CODE REVIEW

**Comment by Eliasvdb** [comment]
cc @patrickhulce

**Comment by Eliasvdb** [comment]
I just signed my CLA, but maybe it needs some time to process. That check should be green soon I guess.

File: docs/recipes/docker-client/Dockerfile [hunk 1]
```diff
@@ -14,6 +17,9 @@ RUN apt-get update \
 RUN npm install -g @lhci/cli@0.8.2
 RUN npm install -g lighthouse
 
+# Install puppeteer
+RUN npm install -g puppeteer
```
Comment (patrickhulce) [inline docs/recipes/docker-client/Dockerfile:21]: you'll need to add this after the next RUN block or it won't be available

`RUN cd /home/lhci/reports && npm link puppeteer`

**Review by patrickhulce — CHANGES_REQUESTED** [review]
thanks @Eliasvdb!

File: docs/recipes/docker-client/Dockerfile (hunk 1 above)
Comment (patrickhulce) [reply docs/recipes/docker-client/Dockerfile:21]: can test it out with a script in `scripts/foo.js` and running `docker run -v "$(pwd)/lhci-data:/home/lhci/reports/.lighthouseci" -v "$(pwd)/scripts:/home/lhci/reports/scripts" lhci-client lhci collect --puppeteerscript=scripts/foo.js --url=https://example.com`

**Comment by Eliasvdb** [comment]
@patrickhulce I added the changes you suggested. I am unable to test it locally with your command though. When I build the docker image and run `docker run -v "$(pwd)/lhci-data:/home/lhci/reports/.lighthouseci" -v "$(pwd)/scripts:/home/lhci/reports/scripts" lhci-client lhci collect --puppeteerScript=scripts/puppeteer-test.js --url=https://example.com` from project root, I get the following error:
```
Failed to move to new namespace: PID namespaces supported, Network namespace supported, but failed: errno = Operation not permitted


TROUBLESHOOTING: https://github.com/puppeteer/puppeteer/blob/main/docs/troubleshooting.md

    at onClose (/usr/local/lib/node_modules/puppeteer/lib/cjs/puppeteer/node/BrowserRunner.js:229:20)
    at ChildProcess.<anonymous> (/usr/local/lib/node_modules/puppeteer/lib/cjs/puppeteer/node/BrowserRunner.js:220:79)
    at ChildProcess.emit (events.js:326:22)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:276:12)
```

I tried setting `--puppeteerLaunchOptions`, like so: `docker run -v "$(pwd)/lhci-data:/home/lhci/reports/.lighthouseci" -v "$(pwd)/scripts:/home/lhci/reports/scripts" lhci-client lhci collect --puppeteerScript=scripts/puppeteer-test.js --puppeteerLaunchOptions="{'args':['--no-sandbox', '--disable-setuid-sandbox']}" --url=https://example.com`, but that isn't working either (same error), am I setting them correctly?

**Comment by patrickhulce** [comment]
> but that isn't working either (same error), am I setting them correctly?

No puppeteerLaunchOptions isn't parsed as a string. I would expect `--puppeteerLaunchOptions.args=--no-sandbox --puppeteerLaunchOptions.args=--disable-setuid-sandbox` to work though.

**Comment by Eliasvdb** [comment]
> I would expect `--puppeteerLaunchOptions.args=--no-sandbox --puppeteerLaunchOptions.args=--disable-setuid-sandbox` to work though

Ah yes, that's it! Seems to work correctly then. Whenever a puppeteerscript is provided it seems puppeteer will need to be launched with both those options. Should those be configured somewhere then instead of leaving it up to the user to specify them? What do you think?

Let me know if you would like to see anything changed, I'll make sure to fix it sooner this time 😃

**Review by patrickhulce — APPROVED** [review]
thanks @Eliasvdb! 🎉

**Comment by patrickhulce** [comment]
> Should those be configured somewhere then instead of leaving it up to the user to specify them? What do you think?

Nah, let's just add a doc example of how to properly invoke it 👍 

CLA issues + an example in the docs and I think we're good to go 🎉

**Comment by Eliasvdb** [comment]
> Nah, let's just add a doc example of how to properly invoke it 👍
I'll make sure to add it!

> CLA issues + an example in the docs and I think we're good to go 🎉
I'm not sure what the problem is, I have signed the CLA with my GitHub username 🤔

**Comment by Eliasvdb** [comment]
@patrickhulce Found and solved the CLA issue! If there's anything else, let me know!

**Comment by Eliasvdb** [comment]
Those failing tests seem unrelated to my change, does it happen more often? Seems timeout related?

**Comment by Djiit** [comment]
Hey team, are we still planning on merging this ? Thanks 🙏

**Pull request merged**